stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -2455,9 +2455,6 @@ void YangParseTreePaths::AddSubtreeInterfaceFromSingleton(
 
   node = tree->AddNode(GetPath("interfaces")(
       "interface", name)("ethernet")("config")("mac-address")());
-  // TODO(tmadejski) Replace the mock value of config MAC address
-  // (0x112233445566) with a value read from the configuration pushed by the
-  // controller once such field is added.
   SetUpInterfacesInterfaceEthernetConfigMacAddress(0x112233445566, node);
```

Comment:
Shouldn't the fixed mac address be changed too? Or do I miss something? (If thats the case just ignore this remark... 😎)

Reply:
I copy these lines to another place and forget to delete them.
Thank you.